.buildkite/pipeline.yml

```diff
@@ -8,7 +8,7 @@ common_params:
         repo: "woocommerce/woocommerce-ios/"
   # Common environment values to use with the `env` key.
   - &common_env
-    IMAGE_ID: xcode-13
+    IMAGE_ID: xcode-13.3.1
```

Comment:
We could probably take the occasion to also update [the value in `.buildkite/cache-builder.yml`](https://github.com/woocommerce/woocommerce-ios/blob/trunk/.buildkite/cache-builder.yml#L16) as well (which was apparently missed last time we updated from 12.5.1 to 13…), to ensure that the cache of the Pods that the project uses (which is build nightly to be shared with each CI job) is build with the same Xcode version as the rest, for consistency.

Reply:
Great point! Done in https://github.com/woocommerce/woocommerce-ios/pull/6884/commits/454c0d79bddbe336d7532693a65b2a6476d804f7